documentation/library_guide/overview.rst

```diff
@@ -42,7 +42,12 @@ requires at least a C++11 compiler. Some APIs of the library may require a highe
 To call Parallel API with the C++ standard policies, you need to install the following software:
 
 * A C++ compiler with support for OpenMP* 4.0 (or higher) SIMD constructs
-* |onetbb_long| or |tbb_long| 2019 and later
+* Depending on what parallel backend you want to use install either:
+  
+  * |onetbb_long| or |tbb_long| 2019 and later
+  * A C++ compiler with support for OpenMP* 4.5 (or higher)
```

Comment:
Is not it enough to use the asterisk only once, when OpenMP is encountered for the first time in a page?

Reply:
According to what I've checked it's enough. Fixed.